corehq/apps/ivr/migrations/0002_call_app_id.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.22 on 2019-10-28 20:52
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+from corehq.apps.ivr.models import Call
+from corehq.apps.app_manager.dbaccessors import get_apps_in_domain
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+app_id_by_form_unique_id = {}
```

Comment:
Why's this in the global scope?  I'm not sure whether it would cause any problems, but it seems unorthodox, at least.

Reply:
I couldn't remember what python passes by value versus by reference, so I was lazy. Fixed in 5dd2681db42

Reply:
Moved this change over to https://github.com/dimagi/commcare-hq/pull/25757/, so now it's 2ace5a83f6cf1259c1d11c793d6c2a2ccf9f847f